src/statespacemodel.jl

```diff
@@ -0,0 +1,141 @@
+immutable StateSpaceModel{T <: Real}
+
+    # Process transition matrix, control matrix, and noise covariance
+    F::Function
+    B::Matrix{T}
```

Comment:
Could the control and feed-forward matrices be functions too? I have not seen this done, but I could imagine situations where it could be useful.